src/paragraph/story.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+import { storiesOf } from '@storybook/react'
+import { withInfo } from '@storybook/addon-info'
+import { text, boolean, withKnobs } from '@storybook/addon-knobs'
+
+import Paragraph from 'paragraph'
+
+const stories = storiesOf('Paragraph', module)
+stories.addDecorator(withKnobs)
+
+stories.add(
+  'Default Paragraph',
+  withInfo('')(() => (
+    <Paragraph
+      expandable={boolean('Expandable', false)}
+      expandLabel={text('Expand label', 'Read more')}
+    >
+      {text('Text', 'This is an example')}
```

Comment:
Would be good if the component was showing a real paragraph by default, here I feel like it's not obvious

Reply:
Yeah no I just mean, put a lot of text here, like a big block of Lorem Ipsum. I don't argue about the tech side, just the visual side :)